python/pyspark/pandas/series.py

```diff
@@ -2154,6 +2159,8 @@ def clip(self, lower: Union[float, int] = None, upper: Union[float, int] = None)
             Minimum threshold value. All values below this threshold will be set to it.
         upper : float or int, default None
             Maximum threshold value. All values above this threshold will be set to it.
+        inplace : bool, default False
+             if True, perform operation in-place
```

Comment:
Maybe do we also might need to `*args` and `**kwargs` for compatibility with pandas ?

<img width="604" alt="Screen Shot 2022-04-05 at 9 16 56 AM" src="https://user-images.githubusercontent.com/44108233/161654114-fb4264c1-a6ce-4eaf-8853-b2cb3a20d31d.png">

Reply:
In our case, additional keywords have no effect always. I'm afraid adding them will be confusing. 

I believe there are other places missing additional keywords as well. How about we discuss them further and resolve them altogether?

Reply:
Sure. Let's keep it as is for now.

Although I suggested to add them just in case, maybe using `*args` and `**kwargs` here and other APIs is fairly uncommon.

Reply:
Sounds good! Thanks!